modules/javafx.graphics/src/main/java/javafx/scene/text/TabStop.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package javafx.scene.text;
+
+/**
+ * This class encapsulates an immutable single tab stop within the {@link TabStopPolicy}.
+ * <p>
+ * A tab stop is at a specified distance from the
+ * left margin, aligns text in a specified way, and has a specified leader.
+ *
+ * @since 999 TODO
+ */
+public class TabStop {
```

Comment:
If this immutable, why not use a `record` instead?

Reply:
good question!

I see no value here in making it a record, and I think there is a possibility for an application to extend TabStop in order to provide additional attributes.

Reply:
(this thing is still in Draft, but thank you for checking up on it!)